packages/flutter_tools/test/integration/flutter_tester_test.dart

```diff
@@ -93,3 +130,16 @@ dependencies:
     sdk: flutter
 ''');
 }
+
+Future<void> _getPackages() async {
+  final List<String> command = <String>[
+    fs.path.join(getFlutterRoot(), 'bin', 'flutter'),
```

Comment:
@devoncarew @scheglov I think this might not be safe on Windows (requires .bat?). Do we have a better way to get the flutter executable?

Reply:
I don't know one.

Reply:
This seems to work - it's how we do it in the `upgrade.dart` command.

Reply:
Hmm, weird - I'll test it to be sure before merging. I'd expect it to work without if `shell: true` but that doesn't seem to be the case in `upgrade.dart`.

Reply:
Hmm, the test is listed in [the AppVeyor](https://ci.appveyor.com/project/flutter/flutter/build/1.0.12504) output and there are no failures, so I guess it's working here too. Maybe it's something weird about nodejs that means I have to use it in Dart Code then.